test/runtests.jl

```diff
@@ -47,6 +47,15 @@ end
         @test isa(ProfileView.view(C=true), ProfileView.Window)
         @test isa(ProfileView.view(fontsize=18), ProfileView.Window)
         @test isa(ProfileView.view(windowname="ProfileWindow"), ProfileView.Window)
+        @test isa(ProfileView.view(graphtype=:icicle), ProfileView.Window)
+
+        @test_logs (:info, "Default graphtype set to \"icicle\"") ProfileView.set_graphtype!(:icicle)
+        @test isa(ProfileView.view(), ProfileView.Window)
+        @test_logs (:info, "Default graphtype set to \"flame\"") ProfileView.set_graphtype!(:flame)
+        @test isa(ProfileView.view(), ProfileView.Window)
```

Comment:
If someone runs these lines locally who has their personal default set to `:icicle`, will this reset their default to `:flame`? If so, let's `try/finally` restore whatever setting the user has.

Reply:
Good catch. Thanks